README.md

```diff
@@ -84,6 +84,16 @@ You can specify this value by `GIT_PR_RELEASE_MENTION` environment variable.
 
 If not specified, the mention will be the PR assignee
 
+## `pr-release.ssl_no_verify`
```

Comment:
It is similer to git's `http.sslVerifier` option
https://git-scm.com/docs/git-config#Documentation/git-config.txt-httpsslVerify